tests/testthat/setup.R

```diff
@@ -0,0 +1,39 @@
+# Download the JDBC drivers used in the tests
+oldJarFolder <- Sys.getenv("DATABASECONNECTOR_JAR_FOLDER")
+Sys.setenv("DATABASECONNECTOR_JAR_FOLDER" = tempfile("jdbcDrivers"))
+
+withr::defer({
+  unlink(Sys.getenv("DATABASECONNECTOR_JAR_FOLDER"), recursive = TRUE, force = TRUE)
```

Comment:
@schuemie I took this from Database connector, but i couldn't seem to find a standard HADES environment variable for this in the R_CMD_check configuration in other packages

Reply:
With the new version of DatabaseConnector we'll have to start downloading the drivers in all unit tests that need to connect  to these databases. We just haven't done that yet in those other packages.

In DatabaseConnector I used the DATABASECONNECTOR_JAR_FOLDER environmental variable to test that behavior. However, here I would recommend not changing any environmental variables. Instead, we can just specify the `pathToDriver` argument when calling `downloadJdbcDrivers()` and `createConnectionDetails()`. So within the `defer` statement we still need to unlink the temp folder to delete the downloaded drivers. We just need a regular variable holding the temp folder name.